<ui version="4.0" >
 <class>QRL_ScopesManager</class>
 <widget class="QDialog" name="QRL_ScopesManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scopes Manager</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QListWidget" name="scopeListWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Scope" >
      <attribute name="title" >
       <string>Scope</string>
      </attribute>
      <widget class="QCheckBox" name="showCheckBox" >
       <property name="geometry" >
        <rect>
         <x>9</x>
         <y>9</y>
         <width>105</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text" >
        <string>Show/Hide</string>
       </property>
      </widget>
      <widget class="QComboBox" name="optionComboBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>180</y>
         <width>115</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>Show Grid</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>set BG Color</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>set Grid Color</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="displayComboBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>240</y>
         <width>150</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>Continuous Roling</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Overwrite</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Trigger +to-</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Trigger -to+</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>hold</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="directionComboBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>210</y>
         <width>105</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>right to left</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>left to right</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="oneshotCheckBox" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>30</y>
         <width>105</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text" >
        <string>One Shot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="triggerPushButton" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>50</y>
         <width>105</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Trigger</string>
       </property>
      </widget>
      <widget class="QwtCounter" name="triggerCounter" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>0</number>
       </property>
       <property name="basicstep" stdset="0" >
        <double>0.100000000000000</double>
       </property>
       <property name="minValue" >
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maxValue" >
        <double>1000000.000000000000000</double>
       </property>
      </widget>
      <widget class="QComboBox" name="traceComboBox_2" >
       <property name="geometry" >
        <rect>
         <x>50</x>
         <y>80</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text" >
         <string>Trace 1</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>190</x>
         <y>150</y>
         <width>55</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text" >
        <string>Filename</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fileLineEdit" >
       <property name="geometry" >
        <rect>
         <x>170</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QwtCounter" name="timeCounter" >
       <property name="geometry" >
        <rect>
         <x>170</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>0</number>
       </property>
       <property name="minValue" >
        <double>0.000000000000000</double>
       </property>
       <property name="maxValue" >
        <double>1000000.000000000000000</double>
       </property>
       <property name="value" >
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="savePushButton" >
       <property name="geometry" >
        <rect>
         <x>170</x>
         <y>240</y>
         <width>75</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Save</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label" >
       <property name="geometry" >
        <rect>
         <x>190</x>
         <y>190</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text" >
        <string>Time [s]</string>
       </property>
      </widget>
      <widget class="QwtCounter" name="dataCounter" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>150</y>
         <width>105</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>1</number>
       </property>
       <property name="basicstep" stdset="0" >
        <double>1.000000000000000</double>
       </property>
       <property name="minValue" >
        <double>1.000000000000000</double>
       </property>
       <property name="maxValue" >
        <double>10000.000000000000000</double>
       </property>
       <property name="stepButton1" >
        <number>100</number>
       </property>
       <property name="stepButton2" >
        <number>1000</number>
       </property>
       <property name="stepButton3" >
        <number>10000</number>
       </property>
       <property name="value" >
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="maxLabel" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>120</y>
         <width>82</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>Data points</string>
       </property>
      </widget>
      <widget class="QwtCounter" name="rrCounter" >
       <property name="geometry" >
        <rect>
         <x>160</x>
         <y>90</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>1</number>
       </property>
       <property name="basicstep" stdset="0" >
        <double>0.100000000000000</double>
       </property>
       <property name="minValue" >
        <double>0.010000000000000</double>
       </property>
       <property name="maxValue" >
        <double>100.000000000000000</double>
       </property>
       <property name="stepButton1" >
        <number>1</number>
       </property>
       <property name="value" >
        <double>20.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="minLabel" >
       <property name="geometry" >
        <rect>
         <x>160</x>
         <y>60</y>
         <width>82</width>
         <height>42</height>
        </rect>
       </property>
       <property name="text" >
        <string>Refresh rate</string>
       </property>
      </widget>
      <widget class="QLabel" name="maxLabel_2" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>10</y>
         <width>55</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text" >
        <string>sec/div</string>
       </property>
      </widget>
      <widget class="QComboBox" name="dxComboBox" >
       <property name="geometry" >
        <rect>
         <x>170</x>
         <y>40</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <property name="minimumContentsLength" >
        <number>1</number>
       </property>
       <item>
        <property name="text" >
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>0.1</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="Trace" >
      <attribute name="title" >
       <string>Trace</string>
      </attribute>
      <widget class="QPushButton" name="traceColorPushButton" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>20</y>
         <width>83</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Color</string>
       </property>
      </widget>
      <widget class="QwtCounter" name="lineWidthCounter" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>100</y>
         <width>74</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>0</number>
       </property>
       <property name="basicstep" stdset="0" >
        <double>1.000000000000000</double>
       </property>
       <property name="minValue" >
        <double>1.000000000000000</double>
       </property>
       <property name="maxValue" >
        <double>50.000000000000000</double>
       </property>
      </widget>
      <widget class="QwtCounter" name="dyCounter" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>170</y>
         <width>74</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>0</number>
       </property>
       <property name="basicstep" stdset="0" >
        <double>0.000100000000000</double>
       </property>
       <property name="minValue" >
        <double>0.000001000000000</double>
       </property>
       <property name="maxValue" >
        <double>1000000.000000000000000</double>
       </property>
       <property name="value" >
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QwtCounter" name="offsetCounter" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>150</y>
         <width>74</width>
         <height>23</height>
        </rect>
       </property>
       <property name="numButtons" >
        <number>0</number>
       </property>
       <property name="basicstep" stdset="0" >
        <double>0.100000000000000</double>
       </property>
       <property name="minValue" >
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maxValue" >
        <double>1000000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="widthLabel" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>80</y>
         <width>64</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>Line Width</string>
       </property>
      </widget>
      <widget class="QLabel" name="offsetLabel" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>130</y>
         <width>64</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>offset</string>
       </property>
      </widget>
      <widget class="QLabel" name="dyLabel" >
       <property name="geometry" >
        <rect>
         <x>180</x>
         <y>150</y>
         <width>64</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text" >
        <string>dy</string>
       </property>
      </widget>
      <widget class="QComboBox" name="traceComboBox" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>20</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QwtWheel" name="offsetWheel" >
       <property name="geometry" >
        <rect>
         <x>30</x>
         <y>190</y>
         <width>64</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
  <customwidget>
   <class>QwtWheel</class>
   <extends>QWidget</extends>
   <header>qwt_wheel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_ScopesManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel" >
     <x>394</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
